pkgs/top-level/all-packages.nix

```diff
@@ -21559,6 +21561,8 @@ with pkgs;
     erlang = erlangR22;
   };
 
+  cyrus = callPackage ../servers/mail/cyrus { };
```

Comment:
The project's website, source repo and doc are named Cyrus IMAP(d).
Even though it supports more than this protocol, we should probably stick
with the same name.